pkgs/development/tools/pip-audit/default.nix

```diff
@@ -43,13 +40,11 @@ buildPythonApplication rec {
     cachecontrol
     cyclonedx-python-lib
     html5lib
-    lockfile
```

Comment:
```
ERROR: Could not find a version that satisfies the requirement lockfile>=0.9; extra == "filecache" (from cachecontrol[filecache]) (from versions: none)
ERROR: No matching distribution found for lockfile>=0.9; extra == "filecache"
```

Reply:
Fixed, I committed the the wrong change.